core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala

```diff
@@ -31,14 +33,11 @@ final case class IntelliJVersion(build: String, release: Option[String]) {
 object IntelliJVersion {
   implicit val configConvert: ConfigConvert[IntelliJVersion] = deriveConvert[IntelliJVersion]
 
-  // TODO(#253): replace Latest with reading from reference.conf
-  val Latest: IntelliJVersion = release("2021.2.1", "212.5080.55")
-
   def snapshot(build: String): IntelliJVersion = {
-    IntelliJVersion(build, None)
+    IntelliJVersion(build = build, release = None, format = Some(configConvert.map(_.format).toString))
   }
 
   def release(version: String, build: String): IntelliJVersion = {
-    IntelliJVersion(build, Some(version))
+    IntelliJVersion(build = build, release = Some(version), format = Some(configConvert.map(_.format).toString))
```

Comment:
why this strange code?

Reply:
_strange_? 🤔

Reply:
@lukaszwawrzyk - please let me know, what exactly is strange / not OK with this change. Named parameters are not OK? Or maybe the fact, that third parameter has been added? Or something else maybe?

Reply:
Why do you need to use configConvert explicitly in the code.

Reply:
It's an easy way to get the `probe.intellij.format` value from the `.conf` file. As we already have this val in the `IntelliJVersion` object - it looks like we can just read this config from the `configConvert` val. Is there something wrong with this?

Reply:
BTW - what is the actual usage of this `implicit val configConvert` here? I mean - before my changes on this PR.